subprojects/packagefiles/openssl/meson.build

```diff
@@ -47,26 +47,17 @@ c_args = []
 asm_opt = get_option('asm')
 
 if asm_opt.disabled()
-  new_gas_or_nasm = false
-  any_gas_or_nasm = false
+  gas_or_nasm = false
 else
-  new_gas_or_nasm = (
-    find_program('as', required: false, version: '>=2.26').found() or
-    find_program('nasm', required: false, version: '>=2.11.8').found()
-  )
-  any_gas_or_nasm = (
-    new_gas_or_nasm or
+  gas_or_nasm = (
     find_program('as', required: false).found() or
     find_program('nasm', required: false).found()
```

Comment:
Shouldn't we keep the minimum versions here?

Reply:
We could, but those versions are already so old, it is not possible that people would have modern enough Python to run Meson at the same time. So I didn't think it was worth polluting the code with additional version checks.